ykrt/src/mt.rs

```diff
@@ -58,8 +59,14 @@ pub struct Location<I> {
     //  │             │ start tracing
     //  │             ▼
     //  │           ┌────────────────────┐
+    //  │           │   PHASE_TRACING    │
+    //  │           └────────────────────┘
+    //  │             │ check for    ▲
+    //  │             │ stuckness    | still
```

Comment:
stuckness :P 

Is that a technical term?

Reply:
Given the space constraints in the diagram "yes" :p